.github/workflows/auto-update.yaml

```diff
@@ -0,0 +1,65 @@
+name: Auto update
```

Comment:
you shouldn't have these in the flathub repo

Reply:
Removed.